src/engine/srv_internal.h

```diff
@@ -79,6 +79,7 @@ struct dss_xstream {
 	bool			dx_main_xs;	/* true for main XS */
 	bool			dx_comm;	/* true with cart context */
 	bool			dx_dsc_started;	/* DSC progress ULT started */
+	bool			dx_progress_started;	/* Network poll started */
```

Comment:
It's not initialized by default. Are you assume 0 == false? Maybe better explicitly init it.

Reply:
It's initialized by D_ALLOC(), I think most DAOS code relies on the calloc() used by D_ALLOC().

Reply:
I understand this but prefer explicit initialization.